src/main/java/edu/hm/hafner/renderer/EchartsSvgRenderer.java

```diff
@@ -0,0 +1,84 @@
+package edu.hm.hafner.renderer;
+
+import edu.hm.hafner.renderer.util.SvgParser;
+import edu.hm.hafner.renderer.util.TriremeResourcesProvider;
+
+import io.apigee.trireme.core.NodeEnvironment;
+import io.apigee.trireme.core.NodeException;
+import io.apigee.trireme.core.NodeScript;
+import io.apigee.trireme.core.ScriptStatus;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Creates a JSON file and passes it as parameter to Trireme. These steps are used to build an SVG using ECharts.
+ * TODO: input Jar file, output to drive
+ * TODO: extract JAR
+ */
+public class EchartsSvgRenderer {
+    private static final Logger LOG = LoggerFactory.getLogger(EchartsSvgRenderer.class);
+
+    //TODO: param1: String options, param2: String parameters
+    private String createSvgString(String[] params) {
+        String svgAsString = "";
+        NodeScript echartsInstance = null;
+
+        try {
+            final String javaScriptFileName = "chartRenderer.js";
+            final String javaScriptFilePath = "/echarts/" + javaScriptFileName;
+            final String nodeModulesPath = "/echarts/node_modules";
+
+            final NodeEnvironment nodeEnv = new NodeEnvironment();
+            final TriremeResourcesProvider triremeResourcesProvider = new TriremeResourcesProvider();
+
+            final File eChartsFile = triremeResourcesProvider.copyJavaScriptFile(javaScriptFilePath);
+            final String triremeWorkingDirectoryPath = triremeResourcesProvider.copyNodeModulesFolder(nodeModulesPath);
+            params[params.length - 1] = triremeWorkingDirectoryPath;
+
+            if (eChartsFile.isFile()) {
+                echartsInstance = nodeEnv.createScript(javaScriptFileName, eChartsFile, params);
+            }
+        } catch (NodeException | IOException e) {
+            LOG.error("Cannot execute ECharts in Trireme due to missing scripts.", e);
+        }
```

Comment:
Logging makes no sense. You need to throw specific exceptions that help the consumer to understand what is going wrong